app/locales/en.json

```diff
@@ -12,26 +12,36 @@
   "Save the following link somewhere, or bookmark it in your browser.": "Save the following link somewhere, or bookmark it in your browser.",
   "That way you can update your report if your health condition changes.": "That way you can update your report if your health condition changes.",
   "I've noted down the link.": "I've noted down the link.",
-  "Proceed":"Proceed",
+  "Proceed": "Proceed",
   "The link has been copied": "The link has been copied.",
   "Could not copy the link, please copy it yourself": "Could not copy the link, please copy it yourself",
   "Contributors": "Contributors",
   "This website is made with help from volunteers. In no particular order:": "This website is made with help from volunteers. In no particular order:",
-  "...and others":"...and others",
-  "The code for the website can be found":"Koden for nettsiden finnes",
+  "...and others": "...and others",
+  "The code for the website can be found": "Koden for nettsiden finnes",
   "here": "here",
-	"About the service": "About the service",
-	"In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.": "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.",
-	"Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to ": "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to ",
+  "About the service": "About the service",
+  "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.": "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.",
```

Comment:
This looks like a single string, but if I understand correctly each line in here sholuld have a `key`: `value`?

Reply:
Jeez, sorry. This is not a single string. Didn't catch the `:` in there!